### PR TITLE
add -wp-ignore-inactive flag which ignores inactive plugin or themes 

### PR DESCRIPTION
**Pull request by kazuminn**
# What did you implement:
add -wp-ignore-inactive flag which ignores inactive plugin or themes

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Move wordpress on linux. Insert the plugin containing the vulnerability and make it active. So run it once and make sure it is detected. Then run it inactive to make sure it is not detected.

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:
You don't have to satisfy all of the following.

- [x] Write tests
- [ ] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [x] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES

### CODE REVIEW

File: wordpress/wordpress.go [hunk 1]
```diff
@@ -260,3 +267,23 @@ loop:
...
+
+func removeInactives(elements models.WordPressPackages) models.WordPressPackages {
+	var inactives []int
+	for i, y := range elements {
+		if y.Status == "inactive" {
+			inactives = append(inactives, i)
+		}
+	}
+	for _, h := range inactives {
+		elements = removeAt(elements, h)
```
Comment (tomoyamachi) [inline wordpress/wordpress.go:279]: This code looks like it does not work well. Slice index will change after `removeAt`.
Please add testcase to contain 2 or more `inactive` status packages.

Maybe following code works well.

```
var removed []models.WpPackage
for _, p := range pkgs {
  if p.Status == "inactive" {
    continue
  }
  removed = append(removed, p)
}
return removed
```

Comment (kazuminn) [reply wordpress/wordpress.go]: fix it.

**Comment by kazuminn** [comment]
@tomoyamachi I actually tested it with wp in action.

 I have a conflict, but I don't know the correct way to write it. Could you fix it?

**Comment by tomoyamachi** [comment]
Go Module is (sometimes) smart. It is easy.

1. `git fetch (upstream or origin) master`
2. `git merge (upstream or origin)/master`
3. delete conflict lines from go.mod and go.sum
4. `go mod tidy`
5. resolve merge conflict

https://stackoverflow.com/questions/16955980/git-merge-master-into-feature-branch

**Comment by kazuminn** [comment]
@tomoyamachi I fixed conflict . and pass ci.

**Comment by kotakanbe** [comment]
LGTM

**Pull request merged**